### PR TITLE
Update PyQt5 bindings

### PR DESCRIPTION
**Pull request by DonnieWest**
Rebased #91 and added the ability to set keyboard cursor shape in Python

### CODE REVIEW

**Comment by DonnieWest** [comment]
Updated this PR with some additional bindings. Should be good to go now :)

File: pyqt/README.md [hunk 1]
```diff
@@ -4,22 +4,22 @@ PyQt5 Bindings for QTermWidget
...
 	$ sudo make install
 If `make install` command will not work just copy the `qtermwidget.so*` files to /usr/lib directory.
-####3. Install PyQt5 and PyQt5-devel if not yet installed.
-####4. Configure, compile and install bindings. Execute in terminal in the qtermwidget bindings folder:
-
-	$ python config.py
+####2. Install PyQt5 and PyQt5-devel if not yet installed.
+####3. Configure, compile and install bindings. Execute in terminal in the qtermwidget bindings folder:
+	$ cd pyqt/
+	$ QT_SELECT=5 python3 config.py
```
Comment (yan12125) [inline pyqt/README.md]: Is python3 necessary? It would be better to have a Python 2/3 compatible ```config.py```

File: pyqt/README.md [hunk 2]
```diff
@@ -4,22 +4,22 @@ PyQt5 Bindings for QTermWidget
 
 INSTALL: 
 ------------
-####1. Download QTermWidget -> https://github.com/lxde/qtermwidget	
 	
-####2. Compile and install it:
+####1. Download, compile and install QTermWidget:
+	$ git clone https://github.com/lxde/qtermwidget.git
```
Comment (yan12125) [inline pyqt/README.md]: Here should be one more ```cd qtermwidget```

**Comment by yan12125** [comment]
Ping @pawelkoston

**Comment by pawelkoston** [comment]
@yan12125 I've added changes You suggested:
https://github.com/pawelkoston/qtermwidget/commit/66b1b70d48f5cc65e5dfaa7947bd0154a1dfad93.
Should I create another pull request?

**Comment by DonnieWest** [comment]
Closing this as #114 is by the original author and contains everything here

**Pull request closed without merging**